src/backend/core/api/serializers.py

```diff
@@ -343,10 +343,10 @@ def validate(self, attrs):
         return attrs
 
 
-class DocumentVersionSerializer(serializers.Serializer):
-    """Serialize Versions."""
+class VersionFilterSerializer(serializers.Serializer):
+    """Validate version filters applied to the list endpoint."""
 
-    etag = serializers.CharField()
-    is_latest = serializers.BooleanField()
-    last_modified = serializers.DateTimeField()
-    version_id = serializers.CharField()
+    version_id = serializers.CharField(required=False, allow_blank=True)
+    page_size = serializers.IntegerField(
+        required=False, min_value=1, max_value=50, default=20
+    )
```

Comment:
`DOCUMENT_VERSIONS_PAGE_SIZE` is at 50, better to put it at 20 maybe, wdyt?

```suggestion
    page_size = serializers.IntegerField(
        required=False, min_value=1, max_value=50, default=settings.DOCUMENT_VERSIONS_PAGE_SIZE
    )
```

Reply:
I have more than 20 pages on my welcome screen and it works great for the moment. Let's decide later and in another PR.